data/scenarios/04Testing/475-wait-one.yaml

```diff
@@ -0,0 +1,43 @@
+name: Wait for one tick
+description: Robots waiting for one tick should not be put into inactive robot queue.
+win: |
+  try {
+    teleport self (0,0);
+    ishere "lambda"
+  } { return false }
+solution: |
+  log "I will win next tick!"; place "lambda";
+robots:
+  - name: base
+    loc: [0,0]
+    dir: [1,0]
+    devices:
+      - logger
+      - grabber
+    inventory:
+      - [1, lambda]
+  - name: sleeper
+    loc: [0,-1]
+    dir: [1,0]
+    devices:
+      - logger
+    program: |
+      log "I shall sleep"; wait 1; log "I have awoken"
```

Comment:
@byorgey does this test your use case enough? I was thinking about one robot `wait 3` and another `wait 1; log "Is it time?"; wait 1` or we could use something from #475 description.

I guess this could also be a normal test, but I wanted to make sure we use the same `gameTick`.

Reply:
At first I was confused what this is testing but then I also saw the extra test code in `integration/Main.hs`.  Yes, I think this is sufficient.